src/nisarqa/processing/insar/quiver_plots.py

```diff
@@ -382,4 +566,282 @@ def truncate_colormap(
     return im, vmin, vmax
 
 
+@dataclass
+class ParamsForAzRgOffsetsToProjected:
+    """
+    Parameters to convert geocoded offsets values from az/range to projected.
+
+    For e.g. GOFF and GUNW, the along track offsets and slant range offsets
+    rasters are geocoded to projected coordinates, but their pixels'
+    values represent offsets in azimuth and slant range directions (in meters).
+    These parameters are needed to rotate/scale the pixel values from
+    azimuth and slant range directions into the X/Y directions of the projected
+    coordinates.
+
+    Parameters
+    ----------
+    epsg : int
+        EPSG of the offsets arrays and coordinate vectors.
+    orbit : isce3.core.Orbit
+        The trajectory of the radar antenna phase center over a time interval
+        that includes the observation times of each point in `geo_pts`.
+    look_side : isce3.core.LookSide or {'left', 'right'}
+        The look direction of the input product (left-looking or right-looking).
+    wavelength : float
+        The radar central wavelength for the frequency subband of the offset,
+        in meters.
+    ground_track_velocity : nisarqa.MetadataLUT3D
+        The input granule's ground track velocity metadata cube.
+    geo2rdr_params : dict of [str, float] or None, optional
+        An optional dict of parameters configuring the behavior of the
+        root-finding routine used in geo2rdr (bracketing implementation).
+        The following keys are supported:
+        'tol_aztime':
+           Azimuth time convergence tolerance, in seconds. Defaults to 1e-7.
+        'time_start':
+           Start of search interval, in seconds. Defaults to `orbit.start_time`.
+        'time_end':
+           End of search interval, in seconds. Defaults to `orbit.end_time`.
+        Defaults to None.
+    """
+
+    epsg: int
+    orbit: isce3.core.Orbit
+    wavelength: float
+    lookside: isce3.core.LookSide | str
```

Comment:
Currently the docstring says "look_side", which I like better

```suggestion
    look_side: isce3.core.LookSide | str
```